app/javascript/components/time_table/timesheet/tag_pill.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
 import { func, bool, string } from 'prop-types';
 
-const TagPill = ({ tag, selected, onClick }) => (
+const classNames = require('classnames');
+
+const TagPill = ({
+  tag, selected, onClick, bold = true,
+}) => (
   <input
-    className={selected ? `tags clickable selected ${tag}` : `tags clickable ${tag}`}
+    className={`${classNames({
```

Comment:
we could just use classNames like following to avoid string interpolation:
classNames(tag, {tags: true, clickable: true, selected, bolded: bold})